features/layer_lock.h

```diff
@@ -82,7 +82,21 @@ void layer_lock_invert(uint8_t layer);
 //   }
 void layer_lock_set_user(layer_state_t locked_layers);
 
+#if LAYER_LOCK_IDLE_TIMEOUT > 0
+
+// If you want to automatically disable layer lock if no keys are pressed after a
+// configurable timeout, define LAYER_LOCK_IDLE_TIMEOUT to a value between 100
+// and 30000 (in milliseconds) in your config.h file and add the following
+// functions as described below.
+
+    void layer_lock_all_off(void);
```

Comment:
How about move the `layer_lock_all_off()` function outside the `#if LAYER_LOCK_IDLE_TIMEOUT > 0` so that it is available regardless of timeout configuration. It does seem like a handy utility function, depending on how people use this library.

Reply:
What do you think about moving `layer_lock_all_off()` outside the `#if`-block so that it is available to users even when there is no timeout? I might have missed your response about this.